render/pod.go

```diff
@@ -96,6 +96,28 @@ var ReplicaSetRenderer = ConditionalRenderer(renderKubernetesTopologies,
 	),
 )
 
+// DaemonSetRenderer is a Renderer which produces a renderable kubernetes daemonsets
+// graph by merging the pods graph and the daemonsets topology.
+var DaemonSetRenderer = ConditionalRenderer(renderKubernetesTopologies,
+	MakeMap(
+		PropagateSingleMetrics(report.Pod),
+		MakeReduce(
+			MakeMap(
+				Map2Parent(report.DaemonSet, "", nil),
+				PodRenderer,
+			),
+			SelectDaemonSet,
+		),
+	),
+)
+
+// ReplicaGroupRenderer is a Renderer which combines the daemonset and deployment views
+// in (for now) a very naive way.
+var ReplicaGroupRenderer = MakeReduce(
+	DeploymentRenderer,
```

Comment:
I am not sure I get this. The ReplicaGroup view will also include DaemonSets? Don't we have separate view for that?

That's confusing ...